src/qt/bitcoingui.cpp

```diff
@@ -846,6 +846,7 @@ void BitcoinGUI::showDebugWindowActivateConsole()
 void BitcoinGUI::showHelpMessageClicked()
 {
     helpMessageDialog->show();
+    helpMessageDialog->raise();//bring to top if obscured
```

Comment:
Not sure if you are aware of `GUIUtil::bringToFront(this)`, seems the right call here.

Reply:
Thanks!